src/pages/MovieDetail/components/CommentList.js

```diff
@@ -0,0 +1,198 @@
+import { React, useState } from 'react';
+// import axios from 'axios';
+import styled from 'styled-components';
+import Modal from './Modal';
+import { SiMega } from 'react-icons/si';
+import { TbPencil } from 'react-icons/tb';
+
+const CommentList = ({ movieInfo }) => {
+  const [commentModal, setCommentModal] = useState(false);
+  const [clickStar, setClickStar] = useState([
+    false,
+    false,
+    false,
+    false,
+    false,
+  ]);
+  const [image, setImage] = useState({
+    image_file: '',
+    preview_URL: 'images/default.png',
+  });
+  const [loaded, setLoaded] = useState(false);
+
+  const openModal = () => {
+    setCommentModal(true);
+  };
+
+  const closeModal = () => {
+    setCommentModal(false);
+  };
+
+  const starRating = index => {
+    let clickStates = [...clickStar];
+    for (let i = 0; i < 5; i++) {
+      clickStates[i] = i <= index ? true : false;
+    }
+    setClickStar(clickStates);
+  };
+  const ratingNum = clickStar.filter(Boolean).length * 2;
+
+  const saveImage = e => {
+    e.preventDefault();
+    const fileReader = new FileReader();
+
+    if (e.target.files[0]) {
+      setLoaded('loading');
```

Comment:
초기값이 false이기 때문에 여기는 true여야 할 것 같습니다!
```suggestion
      setIsLoading(true);
```